docker-storage-setup.sh

```diff
@@ -1169,6 +1169,18 @@ if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
 
+if [ ! -z "$DOCKER_ROOT_LV_SIZE" ]; then
+    Info " DOCKER_ROOT_LV_SIZE is deprecated, please use CONTAINER_ROOT_LV_SIZE"
```

Comment:
This should be `DOCKER_ROOT_VOLUME_SIZE`. There is no variable called `DOCKER_ROOT_LV_SIZE`.

Also can we move all these checks in check_storage_options().

Also this check does not really belong in this patch. Given you moved DOCKER_ROOT_VOLUME_SIZE in docker-storage-setup.sh in patch3, that's where it should be.